src/CachedSchema.php

```diff
@@ -117,7 +117,7 @@ public function buildSchemaCache()
                     new TagDependency(['tags' => $this->cacheTagName])
                 );
 
-                if (YII_ENV == 'dev') {
+                if (YII_ENV_DEV) {
```

Comment:
What about `(defined(YII_ENV) && YII_ENV === 'dev') || (defined(YII_ENV_DEV) && YII_ENV_DEV)`? Or something like this.

Reply:
Redundant check, constant "YII_ENV_DEV" will always be set

https://github.com/yiisoft/yii2/blob/6a04ac9651fe33d264ca913cfa7fffd884f9e86b/framework/BaseYii.php#L40